### DISCUSSION TITLE
Issues with rebar3

### DISCUSSION THREAD

**Discussion by alexdev-at** [gleam-lang/gleam#1589] [Questions & support] [answered]
Hey! A bit desperate atm, so i'm asking you guys here. (I'm on WIndows). I'm using a library which needs rebar3 installed to work. I have it installed, it's added to the path etc. and if u type "rebar3" in the terminal, i get a nice help message, meaning it works. But when i type "gleam run" in my project directory, it gives me this error: https://i.imgur.com/KWKhIGd.png . Idk how to fix it since i've followed all steps, and gleam normally works. Just not when rebar3 is required.

**Comment by lpil** [comment, marked as answer]
Really sorry @ThatOneAlexDev but there's an existing bug ticket for this https://github.com/gleam-lang/gleam/issues/1526

It will be fixed for the next release (soon) but for now you'll need to use WSL or similar. Again very sorry about this unfortunate bug.

**Reply by alexdev-at** [reply]
Thank you so much for the fast reply!
